projects/js-packages/components/package.json

```diff
@@ -20,6 +20,7 @@
 		"@automattic/jetpack-base-styles": "workspace:* || ^0.3",
 		"@babel/core": "7.18.6",
 		"@babel/preset-react": "7.18.6",
+		"@jest/globals": "28.1.0",
```

Comment:
Why do we need this?

Reply:
Because that's imported in `projects/js-packages/components/components/pricing-card/test/component.tsx` and TS won't be happy without installing it as a dependency.